src/style/uomthesis.sty

```diff
@@ -205,18 +205,18 @@
 						\vspace{1cm}
 					ORCID: {\@orcid} \par
 						\vspace{3cm}
-					Submitted in total fulfilment of the requirements of the degree of \par
-					\bigskip
-					{\LARGE Doctor of Philosophy} \par
-				}
+                    Doctor of Philosophy \par

```

Comment:
It looks better to on the final print having "Doctor of Philosophy" in `\LARGE` text.